pkg/master/master_test.go

```diff
@@ -50,7 +121,10 @@ func TestGetServersToValidate(t *testing.T) {
 	}
 }
 
-func TestFindExternalAddress(t *testing.T) {
+// Test_findExternalAddress verifies both pass and fail cases for the unexported
+// findExternalAddress function
+func Test_findExternalAddress(t *testing.T) {
```

Comment:
Not that I know of. However, I felt that ```TestfindExternalAddress``` nor ```TestFindExternalAddress``` was a good representation of that function that was being tested. I don't mind changing it though if there is a convention already around this.